packages/components/popover/src/popover.js

```diff
@@ -0,0 +1,185 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import '@tradeshift/elements.icon';
+import css from './popover.css';
+
+/**
+ * Position on the X axis.
+ */
+export const PositionX = {
+	LEFT: 'left',
+	RIGHT: 'right'
+};
+
+/**
+ * Position on the Y axis.
+ */
+export const PositionY = {
+	TOP: 'top',
+	BOTTOM: 'bottom'
+};
+
+export class TSPopover extends TSElement {
+	static get styles() {
+		return [TSElement.styles, unsafeCSS(css)];
+	}
+
+	static get properties() {
+		return {
+			/** Is the popover visible or not */
+			opened: { type: Boolean, reflect: true },
+			/** Position on X axis relative to the anchor. Could be 'left', 'right' */
+			positionX: { type: String, attribute: 'position-x', reflect: true },
+			/** Position on Y axis relative to the anchor. Could be 'top', 'bottom' */
+			positionY: { type: String, attribute: 'position-y', reflect: true },
+			/** Text in the title */
+			header: { type: String, reflect: true },
+			/** Anchor element for relative positioning. If you need to position absolutely, leave this empty */
+			anchor: { type: String, reflect: true },
+			/** Left offset when popover is positioned absolutely. Use any valid css syntax for the 'left' property */
+			positionLeft: { type: String, attribute: 'position-left', reflect: true },
+			/** Top offset when popover is positioned absolutely. Use any valid css syntax for the 'top' property */
+			positionTop: { type: String, attribute: 'position-top', reflect: true }
+		};
+	}
+
+	constructor() {
+		super();
+		this.opened = false;
+	}
+
+	onClose() {
+		this.opened = false;
```

Comment:
If we are going with [MutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver) it's okay not emit an event for this. But we should either create a helper for it or provide an example with explanation in our documentations, I think it's out of scope of  this PR.

Reply:
Good point, I added a custom event for it.